data/src/main/java/org/apache/iceberg/data/BaseWriterFactory.java

```diff
@@ -220,6 +219,7 @@ protected BaseWriterFactory(Table table, FileFormat dataFileFormat, Schema dataS
         case PARQUET:
           Parquet.DeleteWriteBuilder parquetBuilder = Parquet.writeDeletes(outputFile)
               .setAll(properties)
+              .metricsConfig(metricsConfig)
```

Comment:
nit: Should we leave a todo in the ORC path?

Reply:
actually I guess since we have nothing for the ORC path this is fine

Reply:
Yeah, we have TODOs in other parts to add ORC, so this should be fine.